packages/salesforcedx-vscode-core/package.json

```diff
@@ -24,10 +24,12 @@
   "dependencies": {
     "@salesforce/salesforcedx-sobjects-faux-generator": "42.2.0",
     "@salesforce/salesforcedx-utils-vscode": "42.2.0",
+    "adm-zip": "^0.4.7",
```

Comment:
What is this for and how did we decide to choose this particular implementation? The last release was about three years ago so I'm a bit worried about the support. More importantly, does it work on mac, linux, and windows?

Reply:
I just saw that this is also used by the SFDX CLI so this is probably OK.

Reply:
@vazexqi I did a search and looked at a few options based on recommendations. This one stood out in terms of popularity ([stats here](https://www.npmjs.com/package/adm-zip)), easy of use from api perspective. Although the release is a bit dated, there is recent [commit activity in the repository](https://github.com/cthackers/adm-zip/commits/master), though.

Reply:
This should be OK since the SFDX CLI uses it also.

CC: @amphro